model/eligibility/Eligibility.php

```diff
@@ -138,15 +149,35 @@ public function getId()
     }
 
     /**
-     * @return array
+     * @return Resource
      * @throws \core_kernel_persistence_Exception
      */
+    public function getProctorBypassed(){
+        if($this->proctored === null){
+            $this->proctored =$this->getProperty(EligibilityService::PROPERTY_BYPASSPROCTOR_URI);
+        }
+        return $this->proctored->getUri();
+    }
+
+    /**
+     * @return bool
+     * @throws \core_kernel_persistence_Exception
+     */
+    public function isProctorBypassed()
+    {
+        return  $this->proctored !== EligibilityService::BOOLEAN_TRUE;
```

Comment:
I think you were going to use here `getProctorBypassed()` method which you implemented. Now `$this->proctored` is always `NULL` and API returns `proctored: true` for each eligibility.